test/unit/frontend/github-configuration.test.ts

```diff
@@ -183,6 +187,7 @@ describe("Frontend", () => {
 					.set(
 						"Cookie",
 						getCookieHeader({
+							githubTokenExpiry: Date.now() + 60000,
```

Comment:
How do we know the expiration time is 60000 ? 

(+ We need to make a constant for it)

Reply:
shoot, forgot to push it, this isn't used anymore.